sdk/dotnet/LightSail/Instance.cs

```diff
@@ -196,7 +196,7 @@ public partial class Instance : Pulumi.CustomResource
         public Output<string> PublicIpAddress { get; private set; } = null!;
 
         [Output("ramSize")]
-        public Output<int> RamSize { get; private set; } = null!;
+        public Output<double> RamSize { get; private set; } = null!;
```

Comment:
This is breaking too

Reply:
I’m struggling to have a principled answer on this one - but my feeling is this break is okay for a minor release - due to a combination of (1) rarely used resource (2) break only in .NET and Go and even then only in some cases (3) very easy to adjust code to work and(4) more “correct“ (presumably you are truly allowed to pass fractional ram sizes?

If the last point is not true, then it might make sense to just override this though.